### ISSUE TITLE
Additional Options during rebuild

### ISSUE THREAD

**Issue by Mrjaco12** [Casecommons/pg_search#314]
After digging through the code a bit I found the [additional attributes option](https://github.com/Casecommons/pg_search/blob/master/lib/pg_search/multisearchable.rb#L27) feature for multisearchable. However it appears that `additional_attributes` are not being taken into consideration during a rebuild task. Would it make sense to add a check for them somewhere around [here](https://github.com/Casecommons/pg_search/blob/ef7d88af15ba9b2ba715fd7c45075ad97fab0afd/lib/pg_search/multisearch/rebuilder.rb#L16)? It seems like additional attributes should trigger the `dynamic?` call to return true or something. I can work up a pull request if this would be desirable.

**Comment by eliotsykes** [comment]
Until fix #317 is merged, one possible workaround is to set the `:if` option to a proc that always returns `true`:

```diff
multisearchable against: [:title, :body],
  additional_attributes: -> (thing) { { published_at: thing.published_at } },
+ if: ->(_) { true }  # Workaround additional_attributes issue #314 / PR #317
```

**Comment by nertzy** [comment]
Fixed in https://github.com/Casecommons/pg_search/commit/2b4a2e218be1968cbbfe19a6c0b58eb1222f34d2
